src/main/java/net/apnic/rdap/authority/RDAPAuthority.java

```diff
@@ -279,6 +279,16 @@ public static URI normalizeServerURI(URI server)
         return server;
     }
 
+    public void setDefaultServer(URI defaultServerURI)
+    {
+        if(defaultServerURI == null)
+        {
+            throw new IllegalArgumentException("default server uri cannot be null");
```

Comment:
Looking at this rule and it appears to me that an RDAPAuthority object can be in an "inconsistent" state. ie, if I create it without setting the DefaultServer property, nor add any Servers, then the DefaultServer property can indeed be null.